source/stylesheets/modules/_header.scss

```diff
@@ -110,7 +110,7 @@ $active-nav-color: #1d8feb;
     }
 
     a:focus {
-      color: $black;
+      background-color: $black;
```

Comment:
Minor, but worth making this `inherit`, so it doesn't get a background at all? Easier to change if we decide all headers should be, say, lime green in the future.